block_iagora.php

```diff
@@ -48,17 +48,46 @@ public function get_content() {
 
         $this->content = new stdClass();
         $this->content->footer = '';
-        $this->content->text = '';
+
+        $iframeurl = isset($this->config->iframeurl) ? $this->config->iframeurl : '';
+
+        if (empty($iframeurl)) {
+            $this->content->text = get_string('noiframeurl', 'block_iagora');
+        } else {
+            $this->content->text = '<iframe src="' . $iframeurl . '" width="100%" height="400px" frameborder="0"></iframe>';
+        }
 
         return $this->content;
     }
 
     /**
-     * Defines in which pages this block can be added.
+     * Define in which pages this block can be added.
      *
      * @return array of the pages where the block can be added.
      */
     public function applicable_formats() {
         return ['all' => true];
     }
+
+    /**
+     * Allow the block to be configured.
+     *
+     * @return bool
+     */
+    public function instance_allow_config() {
+        return true;
+    }
+
+    /**
+     * Save instance configuration.
+     * @param stdClass $data The configuration data.
```

Comment:
```suggestion
     * Save instance configuration.
     *
     * @param stdClass $data The configuration data.
```
For consistency as other methods also add a blank line between the title and params)